### PR TITLE
chore(override): add ScaffoldWithCDK func

### PR DESCRIPTION
**Pull request by efekarakus**
Related #4208 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by Lou1415926 — COMMENTED** [review]
The new commit LGTM! Will approve after merging / rebasing 👍🏼 !!

**Review by Lou1415926 — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
lgtm! `.ts`! 😱

**Pull request merged**
